application/ui/widgets/eh-widgets-dir/eh-widgets/src/page/catalog/catalog-tile/CatalogTile.js

```diff
@@ -1,59 +1,99 @@
 
 import "./catalog-tile.scss"
 import CatalogTileOverflowMenu from "./overflow-menu/CatalogTileOverflowMenu"
-import {isHubUser} from "../../../helpers/helpers"
-import {textFromStatus} from "../../../helpers/profiling"
-
-const CatalogTile = ({bundleGroupId, name, description, descriptionImage, categories, status, onAfterSubmit}) => {
-    const [categoryName, setCategoryName] = useState("")
-    useEffect(() => {
-        let isMounted = true;
-        (async () => {
-            const data = await getSingleCategory(categories[0])
-            if (isMounted) {
-                setCategoryName(data.category.name)
-            }
-
-        })()
-
-        return () => {
-            isMounted = false
-        }
-    },[categories])
-
-    const history = useHistory()
-
-    //manage the bundle group detail
-    const handleClick = () => {
-        history.push("/bundlegroup/" + bundleGroupId)
+import { isHubUser } from "../../../helpers/helpers"
+import { textFromStatus } from "../../../helpers/profiling"
+
+const CatalogTile = ({
+  bundleGroupId,
+  name,
+  description,
+  descriptionImage,
+  categories,
+  status,
+  onAfterSubmit,
+}) => {
+  const [categoryName, setCategoryName] = useState("")
+  useEffect(() => {
+    let isMounted = true
+    ;(async () => {
+      const data = await getSingleCategory(categories[0])
+      if (isMounted) {
+        setCategoryName(data.category.name)
+      }
+    })()
+
+    return () => {
+      isMounted = false
     }
+  }, [categories])
 
-    //TODO refactor into an utility function
-    return (
-        <>
-            <div className="CatalogTile">
-                {isHubUser() && <div className="CatalogTile-dropmenu"><CatalogTileOverflowMenu bundleGroupId={bundleGroupId} onAfterSubmit={onAfterSubmit}/></div>}
-                <div onClick={handleClick} className="CatalogTile-card-wrapper">
-                    <div className="CatalogTile-card-icon">
-                        {descriptionImage?<img src={descriptionImage} alt="Logo" width = "45" height = "45"/>:<img src={`${process.env.REACT_APP_PUBLIC_ASSETS_URL}/icon.svg`} alt="Logo"/>}
-                    </div>
-                    <div className="CatalogTile-card-title">{name}</div>
-                    {isHubUser() && <div className="CatalogTile-card-status">{textFromStatus(status)}</div>}
-                    <div className="CatalogTile-card-description">
-                        {description}
-                    </div>
-                    <div className="CatalogTile-card-category">
-                        <Tag type="blue" title="Clear Filter"> {categoryName} </Tag>
-                    </div>
-                </div>
-            </div>
-        </>
+  const history = useHistory()
 
-    )
+  //manage the bundle group detail
+  const handleClick = () => {
+    history.push("/bundlegroup/" + bundleGroupId)
+  }
+
+  let tagColor
+  switch (categoryName) {
+    case "Component Collection":
```

Comment:
@davdet For future reference, hardcoded (and English only) category names like this are really prone to breaking. CC: @GermanoGiudici